packages/rollup-plugin-invariant/src/tests.ts

```diff
@@ -52,8 +52,11 @@ describe("rollup-plugin-invariant", function () {
             if (options && options.errorCodes) {
               assert.strictEqual(node.arguments.length, 2);
               const arg2 = node.arguments[1];
-              recast.types.namedTypes.Literal.assert(arg2);
-              assert.strictEqual(typeof arg2.value, "number");
+              if (recast.types.namedTypes.Literal.check(arg2)) {
+                assert.strictEqual(typeof arg2.value, "number");
```

Comment:
Since all subtypes of `recast.types.namedTypes.Literal` have a `value` property, TypeScript is now totally happy with this line. (Well, TS was happy before, but only because `arg2` had type `any`).